src/util/FactorUtil.js

```diff
@@ -135,6 +135,12 @@ function (Okta, TimeUtil) {
       description: Okta.loc('factor.customFactor.description', 'login'),
       iconClassName: 'mfa-custom-factor',
       sortOrder: 17
+    },
+    'CUSTOM_CLAIMS': {
+      label: '',
+      description: Okta.loc('factor.customFactor.description', 'login'),
+      iconClassName: 'mfa-custom-factor',
+      sortOrder: 18
```

Comment:
Should the sort order be same as GENERIC_SAML since it is going to replace saml?

Reply:
Good point, I can make it to have the same order. Also we are going to remove custom_saml, custom_oidc stuff once the claims_provider goes into EA.

Reply:
then maybe we should have this as `16` and move `GENERIC_SAML` and `GENERIC_OIDC` one up